### PR TITLE
fix dash escape

### PR DESCRIPTION
**Pull request by brenton-at-pieces**
Fixes: https://github.com/ryboe/CSS3/issues/224

Inside of a character range, escape `-` to be treated literally instead of as `[a-z]` range notation in Oniguruma

### CODE REVIEW

**Comment by brenton-at-pieces** [comment]
I'm not sure what the next step for this is. How does this get "workflow approval" ?

**Comment by vrugtehagel** [comment]
It needs the approval of a maintainer to be merged. The maintainers work on this repository voluntarily, so sometimes it can take a little while until they have time to get to it.

That said, I think @argyleink will see this PR soon enough and he'll likely merge it since it's only a small change.

**Comment by brenton-at-pieces** [comment]
Got it; thank you! Didn't know if I needed to do something.

**Review by argyleink — APPROVED** [review]
lgtm!

**Pull request merged**
